pkg/binary/envoy/debug/log_linux.go

```diff
@@ -1,85 +0,0 @@
-// Copyright 2019 Tetrate
```

Comment:
just curious why this file is prefixed `_linux`..? 😄

Reply:
this makes it only compile in linux, which is a compilation bug trap I ran into! you can drift files and it will compile fine since the build skips based on this pattern